js/rpc/definitions/messages.js

```diff
@@ -0,0 +1,41 @@
+export default {
+  rpc: {
+    messages: {
+      getBySchema: {
+        description: "Get messages by schemaId paginated",
+        params: [
+          {
+            name: "schema_id",
+            type: "SchemaId",
+          },
+          {
+            name: "pagination",
+            type: "BlockPaginationRequest",
+          },
+        ],
+        type: "BlockPaginationResponseMessage",
+      },
+    },
+  },
+  types: {
+    BlockPaginationRequest: {
+      from_block: "BlockNumber", // inclusive
```

Comment:
Are `BlockNumber` or `AccountId` types defined? Probably yes.
I tried it with concrete types before like `u32` or `AccountId32` but if these more generic types are defined then it would be awesome.

Reply:
They are! There is an entire collection of supported types, but it is hard to get a list. I found searching in this package to be the easiest way to test to see what was supported: https://github.com/polkadot-js/api/tree/master/packages/types-support